docs/tutorial/nfs-volume-resize.md

```diff
@@ -0,0 +1,166 @@
+# How to Expand/Resize NFS Volume
+
+OpenEBS dynamic-nfs-provisioner **doesn't support** natively expanding NFS volume but we can accomplish the same operation by following a workaround.
+
+
+For expanding NFS volume, you must ensure the following items(prerequisites) are taken care of:
+
+- The BackendStorageClass must support volume expansion, which can configure by editing the StorageClass definition to have `allowVolumeExpansion: true`
+
+  ```
+  kubectl patch sc openebs-lvmpv -p '{"allowVolumeExpansion": true}'
+  ```
+
+  ```sh
+  storageclass.storage.k8s.io/openebs-lvmpv patched
+  ```
+- NFS StorageClass must support volume expansion, which can configure by editing the NFS StorageClass defination to have `allowVolumeExpansion: true`
+
+  ```
+  kubectl patch sc openebs-rwx -p '{"allowVolumeExpansion": true}'
+  ```
+
+  ```sh
+  storageclass.storage.k8s.io/openebs-rwx patched
+  ```
+
+## Steps to perform NFS Volume Expansion
+
+- After meeting the prerequisites, Resize NFS volume by editing backend PVC `spec.resources.requests.storage` to reflect the newly desired size, which must be greater than the original size.
+  ```sh
+  kubectl patch pvc <backend-pvc-name> -n <openebs-namespace> -p '{"spec": {"resources": {"requests": {"storage": "10Gi"}}}}'
+  ```
+  **Note**: OpenEBS dynamic-nfs-provisioner will create backend PVC to provide shared volume and backend PVC has following naming convention `nfs-<application-pv-name>`. Below command will help to get backend PVC name
+  ```sh
+  kubectl get pvc -n <openebs-namespace> -l nfs.openebs.io/nfs-pvc-name=<nfs_pvc_name> -o jsonpath='{.items[0].metadata.name}'
+  ```
+- Wait till success expansion of backend PVC i.e `.status.capacity.storage` should match with `spec.resources.requests.storage`
+  ```sh
+  kubectl get pvc <backend-pvc-name> -n <openebs-namespace>
+  ```
+- Update the NFS PV capacity manually(Since resize is not supported natively by dynamic-nfs provisioner this step will be a workaround)
```

Comment:
If we are doing this, should the value in the NFS PVC also be updated?

Reply:
I thought that if users update the NFS PVC size controller will keep retrying until it got expanded but it is not the case... 

**Events on PVC** Expansion related event occurs only once 

>  Events:
  Type     Reason                 Age                From                                                                                             Message
  ----     ------                 ----               ----                                                                                             -------
  Normal   ExternalProvisioning   24m (x2 over 24m)  persistentvolume-controller                                                                      waiting for a volume to be created, either by external provisioner "openebs.io/nfsrwx" or manually created by system administrator
  Normal   Provisioning           24m                openebs.io/nfsrwx_openebs-nfs-provisioner-75b47f49b4-7fxqp_eda599c9-72ee-41c0-b6f1-b019be1c1508  External provisioner is provisioning volume for claim "default/nfs-pvc"
  Normal   ProvisioningSucceeded  23m                openebs.io/nfsrwx_openebs-nfs-provisioner-75b47f49b4-7fxqp_eda599c9-72ee-41c0-b6f1-b019be1c1508  Successfully provisioned volume pvc-ac15973b-3a91-4832-9a7e-dea7ce88a95c
  Warning  ExternalExpanding      22m                volume_expand                                                                                    Ignoring the PVC: didn't find a plugin capable of expanding the volume; waiting for an external controller to process this PVC.


I will add the step to expand NFS PVC also

Reply:
Update the step to expand NFS PVC